format/puffin-spec.md

```diff
@@ -123,6 +123,44 @@ The blob metadata for this blob may include following properties:
 
 - `ndv`: estimate of number of distinct values, derived from the sketch.
 
+#### `delete-vector-v1` blob type
```

Comment:
These changes are here for reference, but are being discussed separately in #11238.